dali/pipeline/data/tensor_vector.h

```diff
@@ -65,6 +65,21 @@ class TensorVector {
   TensorVector(const TensorVector &) = delete;
   TensorVector &operator=(const TensorVector &) = delete;
 
+  DLL_PUBLIC TensorVector<Backend>(TensorVector<Backend> &&other) noexcept {
+    state_ = other.state_;
+    pinned_ = other.pinned_;
+    tl_ = std::move(other.tl_);
+    type_ = other.type_;
+    views_count_ = other.views_count_.load();
+    tensors_ = std::move(other.tensors_);
+    for (auto &t : tensors_) {
+      if (auto *del = std::get_deleter<ViewRefDeleter>(t->data_)) del->ref = &views_count_;
```

Comment:
I'm not sure if it's possible for `tensors_` to contain nullptrs, but if it is, then
```suggestion
      if (t)
        if (auto *del = std::get_deleter<ViewRefDeleter>(t->data_))
          del->ref = &views_count_;
```

Reply:
Done